diceRoll/diceRollUnitTests.js

```diff
@@ -140,6 +140,52 @@ function runTests(runTF) {
         return !(subRandomIntForDice(testGlobals.sumArray_expanded).includes('d'));
       }),
     ];
+    console.log('DISPLAY FUNCTION TESTS:');
+    passFail += [
+      new UnitTest('toggleSaved()', function() {
+        var result;
+        if (g.contentStatus === content.calculator) {
+          result = (toggleSaved() === content.savedMenu);
+        } else if (g.contentStatus === content.savedMenu) {
+          result = (toggleSaved() === content.calculator);
+        }
+        if (result === true) {
+          toggleSaved();
+        }
+        return result;
+      }),
+      new UnitTest('printToInnerHTML()', function() {
+        var replacePass = true;
+        var addonPass = true;
+        document.getElementById('dispIn').innerHTML = 'init';
+        if (printToInnerHTML('dispIn', 'replaced', true) !== document.getElementById('dispIn').innerHTML) {
+          replacePass = false;
+        }
+        if ('replaced' + printToInnerHTML('dispIn', ' added on') !== document.getElementById('dispIn').innerHTML) {
+          addonPass = false;
+        }
+        return (replacePass && addonPass);
+      }),
+      new UnitTest('clearScreen()', function() {
+        var clearIn = true;
+        var clearOut = true;
+        document.getElementById('dispIn').innerHTML = 'not clear in';
+        document.getElementById('dispOut').innerHTML = 'not clear out';
+        if (clearScreen() !== document.getElementById('dispIn').innerHTML) {
```

Comment:
After calling clearScreen, why are you testing a return value from it? Seems like you should call clearScreen and then check that the screen is clear. This is same as comment below, you are changing State (the DOM is State), so you should be checking the state by itself. (I haven't looked at clearScreen, does it return something normally that you are checking? Maybe you are overloadijng the purpose of clearScreen?)